src/app/frontend/common/components/annotations/annotations.scss

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Since you explained to me that this is a copy of another CSS file (with some changes), I'd prefer to avoid copying that much (for easier maintenance of changes in the future), use the original CSS in your new template, and restrict this CSS file only to the real changes. But you can ignore my feelings if yours are different.